packages/core/src/routes/interaction/utils/blockchain-verification.ts

```diff
@@ -0,0 +1,24 @@
+import type { BlockchainUserInfo } from '@logto/connector-kit';
+import type { BlockchainConnectorPayload } from '@logto/schemas';
+
+import type { WithLogContext } from '#src/middleware/koa-audit-log.js';
+import type TenantContext from '#src/tenants/TenantContext.js';
+
+export const verifyBlockchainIdentity = async (
```

Comment:
📝 We don't really do anything here... There isn't any more identity to verify beyond their address. Probably can just remove from blockchain flow